src/components/Auth/AuthConfig.tsx

```diff
@@ -0,0 +1,179 @@
+import { loader } from './assets/loader'
+import { success } from './assets/success'
+import { borderRadius, color } from '@onfido/castor'
+import { FaceTecCustomization } from '~auth-sdk/FaceTecSDK.js/FaceTecCustomization'
+import { FaceTecSDK } from '~auth-sdk/FaceTecSDK.js/FaceTecSDK'
+import { UICustomizationOptions } from '../../types/ui-customisation-options'
+
+const ConfigBuilder = () => {
+  const BaseURL = process.env.AUTH_URL
+  return {
+    BaseURL,
+    getAuthCustomization,
+  }
+}
+
+const getAuthCustomization = (
+  dimMode: boolean,
+  customUI?: UICustomizationOptions
+) => {
+  const uiDefaults = {
```

Comment:
Minor optimisation tweak here: if `uiDefaults` won't be dynamically manipulated during run-time, you might want to declare it outside of this method.

Reply:
uiDefaults is dependent on the `dimMode` boolean to set the values, so we should leave it here, I think?